2.2/.sti/bin/assemble

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function rake_assets_precompile() {
+  [ -n "$DISABLE_ASSET_COMPILATION" ] && return
+  [ ! -f Gemfile ] && return
+  [ ! -f Rakefile ] && return
+  ! grep " rails " Gemfile.lock >/dev/null && return
+  ! grep " execjs " Gemfile.lock >/dev/null && return
+  ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
+
+  echo "---> Starting asset compilation."
```

Comment:
We are trying to make these messages consistent, so when a user look into build logs they will make sense :-)
The standard we are following is that if you're reporting progress, append ` ...` at the end (with space) (eg. `---> Starting asset compilation ...`)